hw1/02-palindrome.js

```diff
@@ -1,3 +1,25 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
+
+function handleInput() {
+  const value = parseFloat(elem.value);
+  const resultDiv = document.querySelector("#result");
+
+  if (value >= 0) {
+    const numString = value.toString();
+    const reverseString = numString.split("").reverse().join("");
+
+    if (numString == reverseString) {
```

Comment:
This can be a === instead of == (15.1 of Airbnb JavaScript Style Guide).

Reply:
does both work same?

Reply:
In this case yes because both are strings. The only difference between === and == is that for === they have to both be the same type. == can be used as well, it just allows for different types, but it is better to use === if there are of same type.

Reply:
Ah I see, thank you. I'll go ahead and update that!